history.txt

```diff
@@ -1,19 +1,51 @@
 OMERO version history
 =====================
 
+5.3.2 (May 2017)
+----------------
+
+This is a bug-fix release.
+
+Improvements include:
+
+- improved populate_metadata plugin
+- fixed deletion of a range of objects from CLI
+- textual annotations without a namespace can now be added at import using the CLI
+- improved thumbnails retrieval in OMERO.web
+- added "Open With" option to the right-hand panel in OMERO.web
+- group owners are now prevented from annotating other people's data in OMERO.web
+- preview of wells now available in the right-hand panel
+
+Sysadmin changes include:
+
+- made the Django middleware classes configurable using a new property
+- added property to allow connections from specified origins (CORS)
+- administrators can move data between groups without belonging to source and target groups using the CLI
+- for OMERO.web apps to be available via "Open With" option, administrators need to use
+the "omero.web.open_with" configuration option
```

Comment:
If this is a new line, you'll need to indent it to be part of the bullet point or the docs build will complain